src/components/learner-credit-management/cards/AssignmentModalSummary.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Stack } from '@edx/paragon';
+
+import { formatPrice } from '../data';
+import AssignmentModalSummaryEmptyState from './AssignmentModalSummaryEmptyState';
+import AssignmentModalSummaryLearnerList from './AssignmentModalSummaryLearnerList';
+
+const AssignmentModalSummary = ({
+  course,
+  learnerEmails,
+}) => {
+  const learnerEmailsCount = learnerEmails.length;
+  const hasLearnerEmails = learnerEmailsCount > 0;
+
+  let summaryHeading = 'Summary';
+  if (hasLearnerEmails) {
+    summaryHeading = `${summaryHeading} (${learnerEmailsCount})`;
+  }
+  return (
+    <>
+      <h5 className="mb-4">{summaryHeading}</h5>
+      <Stack gap={2.5}>
+        <Card className="rounded-0 shadow-none">
+          <Card.Section className="py-2">
+            {hasLearnerEmails ? (
+              <AssignmentModalSummaryLearnerList
+                course={course}
+                learnerEmails={learnerEmails}
+              />
+            ) : (
+              <AssignmentModalSummaryEmptyState />
+            )}
+          </Card.Section>
+        </Card>
+        {hasLearnerEmails && (
+          <Card className="rounded-0 shadow-none">
+            <Card.Section className="d-flex justify-content-between py-2">
+              <div>Total assignment cost</div>
+              <div>{formatPrice(learnerEmailsCount * course.normalizedMetadata.contentPrice)}</div>
```

Comment:
[nit] Similar to the nit mentioned before. Labeling the business logic would help to give some context on what the value means for debugging purposes.

```
const totalCustomerCost =  learnerEmailsCount * course.normalizedMetadata.contentPrice
.
.
<div>{formatPrice(totalCustomerCost)}</div>
```

Naming conventions totally up to you if you decide to move forward with the suggested change